### ISSUE TITLE
BLUAlert not reacting to Dimensional Death in Fei'yin

### ISSUE THREAD

**Issue by Cellcubed** [Windower/Lua#1968]
While trying to learn dimensional death in fei'yin from specters, BLUAlert did not play a sound or print a message to the chat log when the specters used the ability.

**Comment by RubenatorX** [comment]
You're certain that you didn't already secretly learn it?

**Comment by Cellcubed** [comment]
Yes, I am certain. I would not have posted this otherwise. I didn't know the spell. The specters used the ability several times. The addon did not react. I then saw the message in the chat log about having learned the spell. I then went to learn a different spell. The addon reacted properly to the different spell.

**Comment by KenshiDRK** [comment]
thats cause blualert only check monter_abilities.lua for it and fomor abilities are in weapon_skills.lua, if the ability id the mob uses is below 256 it should check in weapon_skill.lua and if it is above 256 it should check monster_abilities.lua.
